### DISCUSSION TITLE
lookAt not working

### DISCUSSION THREAD

**Discussion by swafe** [google/filament#7716] [Q&A]
When onClick the view should change based on lookAt(). But when I click nothing is changing in the surfaceView while when i uncomment setScaling the view is scaling as expected. Any suggetions?

 ```
reset_filament_view.setOnClickListener {
            modelViewer.camera.lookAt(
                1.0, 100.0, 1.0,
                0, 0, 0,
                0.0, 10.0, 0.0)

//            modelViewer.camera.setScaling(1.80,1.80)
        }
```

**Comment by bastionkid** [comment]
> When onClick the view should change based on lookAt(). But when I click nothing is changing in the surfaceView while when i uncomment setScaling the view is scaling as expected. Any suggetions?
> 
> ```
> reset_filament_view.setOnClickListener {
>            modelViewer.camera.lookAt(
>                1.0, 100.0, 1.0,
>                0, 0, 0,
>                0.0, 10.0, 0.0)
> 
> //            modelViewer.camera.setScaling(1.80,1.80)
>        }
> ```

As @prideout replied in one of the other thread your `lookAt` might be getting overridden by the `ModelViewer.render()`function.
Ref: https://github.com/google/filament/issues/1062#issuecomment-824189555
